kuma/static/js/helpfulness.js

```diff
@@ -3,8 +3,13 @@
     // this feature requires localStorage
     if (('localStorage' in win)) {
         var ignore = localStorage.getItem('helpful-ignore') === 'true'; // true if ever clicked ignore
-        var askedRecently = parseInt(localStorage.getItem(doc.location + '#answered-helpful'), 10) > Date.now();
-        if (!ignore && !askedRecently) {
+        var articleAskedRecently = parseInt(localStorage.getItem(doc.location + '#answered-helpful'), 10) > Date.now();
+        var lastAsked = localStorage.getItem('helpful-asked-last');
+        var today = new Date();
+        today.setHours(0,0,0,0);
+        var askedToday = String(lastAsked) === String(today);
+
+        if (!ignore && !articleAskedRecently && !askedToday) {
             // ask about helpfulness after 1 min
             setTimeout(inquire, 60000);
```

Comment:
Not part of your PR but I think we should move `60000` up in the file as a var since it's a "setting", so to speak.